docs/technical_reference/property_models/NaCl_T_dep.rst

```diff
@@ -1,7 +1,7 @@
 NaCl Property Package with Temperature Dependence
 =================================================
 
-This package implements property relationships for an NaCl solution.
+This package implements property relationships for a NaCl solution.
```

Comment:
```suggestion
This package implements property relationships for an NaCl solution.
```
I've always gone with reading this as "an N-a-c-l solution".

Reply:
So, it depends on whether we want to consider this a formal or informal use case because technically, according to ACS, the full element name should be used when choosing whether to use a or an. 
(Informal: "an N-a-c-l solution" or Formal:  "a sodium chloride solution")

Reply:
I'm good with either "an NaCl solution" or "a sodium chloride solution", so long as it is _not_ changed to "a NaCl solution".